src/NUnitEngine/nunit.engine/Services/SettingsService.cs

```diff
@@ -1,50 +0,0 @@
-// Copyright (c) Charlie Poole, Rob Prouse and Contributors. MIT License - see LICENSE.txt
-
-using System;
-using System.IO;
-using NUnit.Engine.Internal;
-
-namespace NUnit.Engine.Services
-{
-    /// <summary>
-    /// Summary description for UserSettingsService.
-    /// </summary>
-    public class SettingsService : SettingsStore, IService
-    {
-        private const string SETTINGS_FILE = "Nunit30Settings.xml";
-
-        public SettingsService(bool writeable)
-            : base(Path.Combine(NUnitConfiguration.ApplicationDirectory, SETTINGS_FILE), writeable) { }
```

Comment:
I think `NUnitConfigurations.AppDirectory` can also be removed now - it was only used to store settings, I think? 🙂